client/state/themes/selectors.js

```diff
@@ -239,6 +239,23 @@ export function isRequestingTheme( state, siteId, themeId ) {
 }
 
 /**
+ * Returns true if a request is in progress for the site active theme, or
+ * false otherwise.
+ *
+ * @param  {Object}  state  Global state tree
+ * @param  {Number}  siteId Site ID
+ * @return {Boolean}        Whether request is in progress
+ */
+export function isRequestingActiveTheme( state, siteId ) {
+	const activeThemeRequests = get( state, 'themes.activeThemeRequests', null );
+	if ( ! activeThemeRequests ) {
+		return false;
+	}
+
+	return get( activeThemeRequests, siteId, false );
```

Comment:
I think we could drop everything above this line and change it to

```js
return get( state.themes.activeThemeRequests, siteId, false );
```

since `state.themes.activeThemeRequests` is guaranteed to always be there (if no entry for any `siteId` was ever added, it will still fall back to the reducer's [`initialState`](https://github.com/Automattic/wp-calypso/blob/master/client/state/themes/reducer.js#L67), which is an empty object).